Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM eclipse-temurin:17-jdk-alpine
+VOLUME /tmp
+COPY build/libs/*-SNAPSHOT.jar app.jar
```

Comment:
Building the files and then copying to the container will make sure we pick the latest changes.

Reply:
I agree, made the changes.